config/initializers/set_release_footprint.rb

```diff
@@ -1,2 +0,0 @@
-# We will set RELEASE_FOOTPRINT in our Forem Cloud environment, or use HEROKU_SLUG_COMMIT if set (e.g. Heroku env)
-ENV["RELEASE_FOOTPRINT"] ||= ENV["HEROKU_RELEASE_CREATED_AT"]
```

Comment:
**Confirmation required:** I removed this because we have explicitly set the variables in places where we use them. I could totally be missing an important part of the code, hence it would be helpful to get other developer eyes on this.

Reply:
seems like dead code to me.